packages/charts/src/components/portal/portal.component.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
`import { Portal } from '@keen.io/ui-core';`

there's no need to create another portal

Reply:
We need to use portal ref here instead of id as a string - to be sure that portal is rendered before passing tooltip content. Im not sure if extending logic of Portal from ui-core is a good idea because it has "modalContainer" fieldname which may be confusing to use with tooltips.